### DISCUSSION TITLE
pb integration with frontend examples

### DISCUSSION THREAD

**Discussion by elycheikhsmail** [pocketbase/pocketbase#558] [Q&A] [answered]
I hope there some basic example of pb integrations with frontend (svelte or vue or another framework don't matter ) in examples/todos in pb repos, this we will make using and understanding how pb more easy for frontend dev

**Comment by ganigeorgiev** [comment]
There are plans to add some simple examples in the documentation (alongside a new "guides/cookbook" section), but that is a low priority right now.

If you've stumbled on something something specific, feel free to comment here (or open a new discussion) and I'll try to help.

_I'll move the issue to Q&A discussion._

**Comment by clacorte** [comment, marked as answer]
There is a nice example in this link (vue3, typescript and vite): https://studioterabyte.nl/en/blog/pocketbase-vue-3
